src/Symfony/Component/Routing/RouteCompiler.php

```diff
@@ -29,11 +29,19 @@ class RouteCompiler implements RouteCompilerInterface
     const SEPARATORS = '/,;.:-_~+*=@|';
 
     /**
+     * The maximum supported length of a PCRE subpattern name
+     * http://pcre.org/current/doc/html/pcre2pattern.html#SEC16.
+     *
+     * @var int
+     */
+    const VARIABLE_MAXIMUM_LENGTH = 32;
```

Comment:
please add `@internal` also so we don't have to apply the bc policy on this constant